code/__DEFINES/citadel_defines.dm

```diff
@@ -38,9 +38,9 @@
 #define COCK_SIZE_MIN		1
 #define COCK_SIZE_MAX		20
 
-#define COCK_GIRTH_RATIO_MAX		1.25
-#define COCK_GIRTH_RATIO_DEF		0.75
-#define COCK_GIRTH_RATIO_MIN		0.5
+#define COCK_GIRTH_RATIO_MAX		.42
```

Comment:
it needs to be 0.42 and so on

Reply:
did it not fucking commit the changes GOD I LOVE WEBEDITS.

"commit directly to branch" my ass 😂

Reply:
oh no I just left out the zeros whoops